test/support/utils.h

```diff
@@ -792,14 +795,16 @@ transform_reduce_serial(InputIterator first, InputIterator last, T init, BinaryO
     return init;
 }
 
-static const char*
-done()
+int
+done(bool is_passed = true)
 {
-#if _PSTL_TEST_SUCCESSFUL_KEYWORD
-    return "done";
-#else
-    return "passed";
-#endif
+    if(is_passed){
+        ::std::cout <<"Passed\n";
+        return 0;
+    }else{
+        ::std::cout <<"Skipped\n";
```

Comment:
I suggest writing additional info, f.e.,  "<id> group tests are skipped" or something like that.

Reply:
When we run tests with cmake, it prints all skipped tests. If we run one test other way, then we know, why it is skipped. Could you describe case when we need additional info?

Reply:
Discussed it and decided to keep it in mind on future updates.